libbeat/scripts/Makefile

```diff
@@ -305,16 +306,8 @@ index-template: ## @build Generate index templates for the given $VERSION. This
 ### KIBANA FILES HANDLING ###
 ES_URL?=http://localhost:9200
 
-.PHONY: export-dashboards
-export-dashboards: python-env update
-	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/dev-tools/export_dashboards.py --url ${ES_URL} --dir $(shell pwd)/_meta/kibana --regex ${BEAT_NAME}-*
```

Comment:
It's a bit tricky for `export-dashboards` as we need to pass the list of ids that we want to export in one json file. Usually you would run it to export the dashboards for a module, not for a Beat. Added to the TODO list in the meta issue: https://github.com/elastic/beats/issues/4409